ilc/config/custom-environment-variables.json5

```diff
@@ -15,5 +15,6 @@
     },
     newrelic: {
         licenseKey: 'NR_LICENSE_KEY',
-    }
+    },
+    trustDomains: 'TRUST_DOMAINS'
```

Comment:
overrideConfigTrustedOrigins
OVERRIDE_CONFIG_TRUSTED_ORIGINS

add new var to default.json5